force-app/main/default/classes/AccountHelper.cls

```diff
@@ -28,19 +28,54 @@ Always ensure that your helper methods are bulk-safe to handle multiple records
 */
 public with sharing class AccountHelper {
     
-    public void setTypeProspect(SObject objects){
+    public static void updateAccountFileds(List<Account> newAccounts) {
+        List<Contact> contacts = new List<Contact>();
 
-    }
+        for (Account acc : newAccounts){
+
+            //Set the account type to prospect.
+            //When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
+            if(String.isBlank(acc.Type)) {
+                acc.Type = 'Prospect';
+            }            
+            
+
+            //Set the account rating to hot.
+            //When an account is inserted set the rating to 'Hot' if the Phone, Website, and Fax ALL have a value.
+            if(String.isNotBlank(acc.Phone) && String.isNotBlank(acc.Fax) && String.isNotBlank(acc.Fax)) {
+                acc.Rating = 'Hot';
+            }
```

Comment:
This should really be two methods in my opinion. You are doing two things setting the Type field and then setting the rating. 

As more business requirements come, if you keep adding logic to this method, it will get harder and harder to test and maintain. 

This is what my triggerhandler method looks like. 

```
 public override void beforeInsert(){
        setDefaultAccountType();
        setDefaultBillingAddress();
        setRating();
    }
    ```